daml-lf/archive/da/daml_lf_1.proto

```diff
@@ -528,6 +528,17 @@ message Expr {
     Expr variant_arg = 3;
   }
 
+   // Enum construction ('ExpEnumCon')
```

Comment:
add the note about enums at the top of this file

Reply:
done